runtime/include/chpl-tasks-prvdata.h

```diff
@@ -23,7 +23,14 @@
 // This header file provides chpl_comm_taskPrvData_t
 #include "chpl-comm-task-decls.h"
 
-// The type for task private data
+// Type for Chapel-managed task private data
+// to be copied to new tasks.
+// Includes 'serial_state'
+typedef struct {
+  unsigned char data[32];
```

Comment:
@gbtitus - I already added the check here:
https://github.com/chapel-lang/chapel/pull/6765/files#diff-d5ae6549828c154a2a8864af5cf85f21R118

I have a feeling you havn't looked at that file (ChapelTaskData.chpl) yet. Please add a comment on that part if you still feel we need such a comment. I personally think the assert there is enough.